google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1821,12 +1905,29 @@ def iter(self):
                 except ResourceNotSupported as e:
                     # API client doesn't support this resource, ignore.
                     LOGGER.debug(e)
+                except Exception as e:  # pylint: disable=broad-except
```

Comment:
why is this needed?

Reply:
Broad except is needed to catch all exceptions other than ResourceNotSupported while iterating through the org policies.

Reply:
This was not here before, we do have other exception handling on a higher level.

Reply:
I later realized this and had uncommented the broad exception. Removed now.